ddtrace/llmobs/_writer.py

```diff
@@ -141,6 +141,15 @@ def _url(self) -> str:
     def _data(self, events: List[Any]) -> Dict[str, Any]:
         raise NotImplementedError
 
+    def recreate(self):
+        # type: () -> BaseLLMObsWriter
```

Comment:
Type hinting is directly in the function signature now
```suggestion
    def recreate(self) -> BaseLLMObsWriter:
```